### PR TITLE
Tensorflow 2 upgrade

### PR DESCRIPTION
**Pull request by monz**
Add compatibility to Tensorflow 2.x

### CODE REVIEW

**Comment by monz** [comment]
@googlebot I signed it!

File: requirements.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@ matplotlib==2.2.4
 Pillow==6.2.0
 scikit-learn==0.20.3
 scipy==1.2.1
-tensorflow==1.15.2
+tensorflow==2.3.0
```
Comment (jameswex) [inline requirements.txt:5]: just merged in change to alter this dep to 2.4.0 so you can remove this line from this PR and i'll get it in.

**Review by jameswex — COMMENTED** [review]
sorry i never got around to this, please take a look at my one comment

File: requirements.txt (hunk 1 above)
Comment (monz) [reply requirements.txt]: ok, just updated the PR, should be fine now

**Comment by monz** [comment]
> sorry i never got around to this, please take a look at my one comment

no worries :)

**Review by jameswex — APPROVED** [review]

**Pull request merged**